src/map/Map.js

```diff
@@ -611,6 +612,8 @@ L.Map = L.Evented.extend({
 		if (this.options.trackResize) {
 			L.DomEvent[onOff](window, 'resize', this._onResize, this);
 		}
+
+		this[onOff]('moveend', this._onMoveEnd);

```

Comment:
Oh, no, I mean only move the condition, not the filter itself. Don't put a moveEnd listener if there's no point.